app/views/server_load_readings/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= render "servers/heading", server: @server, tab: "readings" %>
+<div class="row">
+  <div class="col-md-12">
+    <div class="body">
+      <div class="chart" id="server_load_history">
+        <svg></svg>
+      </div>
+    </div>
+  </div>
+</div>
+
+<style media="screen">
+  .chart svg {
+    height: 250px;
+  }
+</style>
+
+<script type="text/javascript">
+  $(document).on('turbolinks:load', function() {
```

Comment:
It's a very specific script for the graph on this page with labels, ranges urls etc. Don't think it will be a big improvement or reused again.